mm2src/coins/z_coin.rs

```diff
@@ -890,7 +924,7 @@ impl<'a> UtxoCoinBuilder for ZCoinBuilder<'a> {
                 .await?
             },
         };
-
+        let z_tx_prover = self.z_tx_prover().await?;
```

Comment:
Why move this from its original place? please move this back to it's original place after https://github.com/KomodoPlatform/atomicDEX-API/blob/418f07410dfc6a62d6864e0f48df3a9c488def8c/mm2src/coins/z_coin.rs#L879-L884 it's good to check for zcash params before initialized DBs etc..

Reply:
right thanks, done